docs/docs/1.3.1/community/data-collection.md

```diff
@@ -0,0 +1,55 @@
+# Kuma data collection
+
+By default, Kuma collects some data about your deployment when you install. This data is sent to Kong servers.
+
+## Disable data collection on Kubernetes
+
+Set the following environment variable:
+
+```
+KUMA_REPORTS_ENABLED=false
+```
+
+:::: tabs
+::: tab "kumactl"
+
+Pass the environment variable to the `--env-var` argument when you install:
+
+```shell
+kumactl install control-plane \
+  --env-var KUMA_REPORTS_ENABLED=false
+```
+
+:::
+::: tab "Helm"
+
+Set the environment variable:
+
+```shell
+helm install --version 0.7.1 --namespace kuma-system \
+  --set controlPlane.envVars.KUMA_REPORTS_ENABLED=false \
+   kuma kuma/kuma
+```
+
+:::
+::::
```

Comment:
I created `configuration` section for this specific reason. I noticed that we are describing everytime how to configure Kuma. I think what we should do from now one is to say that you need to set `KUMA_REPORTS_ENABLED=false` and point user to configuration for them to learn how to set it